Neos.Flow/Classes/Command/PackageCommandController.php

```diff
@@ -78,10 +78,11 @@ public function injectBootstrap(Bootstrap $bootstrap)
      * @Flow\FlushesCaches
      * @param string $packageKey The package key of the package to create
      * @param string $packageType The package type of the package to create
+     * @param string $packagesPath The path where new packages are created relative to the flow root path
      * @return void
      * @see neos.kickstarter:kickstart:package
      */
-    public function createCommand(string $packageKey, string $packageType = PackageInterface::DEFAULT_COMPOSER_TYPE)
+    public function createCommand(string $packageKey, string $packageType = PackageInterface::DEFAULT_COMPOSER_TYPE, ?string $packagesPath)
```

Comment:
What do you think about setting 'DistributionPackages' as default here and use it if it exists? Would be a better default.

Reply:
I removed the change in this file and detect the DistributionPackages folder.